### PR TITLE
Allow log2() to work on large integers

### PR DESCRIPTION
**Pull request by devonhollowood**
Part of #1671

My plan is to extend this to log() and log10() in the next PR — I wanted to see how you felt about the approach, first. Let me know what you think.

### CODE REVIEW

**Comment by devonhollowood** [comment]
It looks like the error is in `test_statistics.py`:

```
UNEXPECTED SUCCESS: test_regression_25177 (__main__.TestMean)
```

However, I'm not sure why this is suddenly passing, so I'm a little hesitant to remove the `@unittest.expectedFailure`. What do you think?

File: stdlib/src/math.rs [hunk 1]
```diff
@@ -155,13 +156,35 @@ mod math {
         }
     }
 
+    /// Generates the base-2 logarithm of a BigInt `x`
+    fn ilog2(x: &BigInt) -> f64 {
```
Comment (youknowone) [inline stdlib/src/math.rs:160]:
```suggestion
    fn int_log2(x: &BigInt) -> f64 {
```

**Review by youknowone — CHANGES_REQUESTED** [review]
Great! I left a naming style request.

Yes, removing the decorator is correct way. Maybe it was using log2 somewhere and you caught it.

File: stdlib/src/math.rs (hunk 1 above)
Comment (devonhollowood) [reply stdlib/src/math.rs]: Done and done!

**Review by youknowone — APPROVED** [review]
Thank you for contributing!

**Pull request merged**